bundle.Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM scratch
```

Comment:
since these files are autogenerated, could you please document the steps needed to generate them? If it's all provided by operator-sdk even better, please just state so (with a commandline example if possible) in the commit message and we're good to go.

Reply:
done